docs/cristin.ttl

```diff
@@ -476,7 +476,7 @@
 :TYPE_BOK_RAPPORT
     a               :cristinfield ;
     rdfs:comment    """
-    A type of book called rapport.
+    UNKNOWN. This value can possibly be null be it is not known what conditions dictates it
```

Comment:
Is this a report about the metadata of the  book or is it metadata about a book that is a report? A quick look in the export data shows that the following types have such a field
```
"DRGRADAVH"
"POPVIT_BOK"
"OVERSETTELSE_BOK"
"MASTERGRADSOPPG"
"KOMMENTARUTG"
"HOVEDFAGSOPPGAVE"
"RAPPORT"
"LEKSIKON"
"MAGISTERAVH"
"KOMPENDIUM"
"FOREDRAG_FAG"
"LÆREBOK"
"ARTIKKEL_POP"
"FAGBOK"
"UTSTILLINGSKAT"
"OPPSLAGSVERK"
"LISENSIATAVH"
"FORSKERLINJEOPPG"
"MONOGRAFI"
"ANTOLOGI"
```

Reply:
FOREDRAG_FAG og ARTIKKEL_POP kan ikke finnes i bok_rapport. Korrekte kategorier er:
ANTOLOGI
DRGRADAVH
FAGBOK
FORSKERLINJEOPPG
HOVEDFAGSOPPGAVE
KOMMENTARUTG
KOMPENDIUM
LEKSIKON
LISENSIATAVH
LÆREBOK
MAGISTERAVH
MASTERGRADSOPPG
MONOGRAFI
OPPSLAGSVERK
OVERSETTELSE_BOK
POPVIT_BOK
RAPPORT
UTSTILLINGSKAT